dysnix/heimdall/templates/service-metrics.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.services.metrics.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "heimdall.fullname" . }}-metrics
+  {{- if .Values.services.metrics.annotations }}
```

Comment:
@voron 
I thought on this, but it may be a better solution to use construction like in ServiceMonitor template, f.e.
```
      {{- with (omit .Values.services.metrics "enabled" "ports" "selector") }}
      {{- toYaml . | nindent 6 }}
      {{- end }}
```
WDYT on this?

Reply:
My idea is to specify `.Values.services.metrics` just once per `service-metrics.yaml`.  
I don't think we need to move all the service options to values. It's hard to keep a balance between :)